samples/MASMessagingSample/build.gradle

```diff
@@ -33,10 +33,13 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    testCompile 'junit:junit:4.12'
     compile "com.android.support:recyclerview-v7:$rootProject.ext.recyclerViewVersion"
     compile "com.android.support:design:$rootProject.ext.designVersion"
+    compile 'pl.droidsonroids.gif:android-gif-drawable:1.2.7'
```

Comment:
If you're still in need of an icon, I'd highly recommend https://materialdesignicons.com/ as it includes Google's own Material icons, as well as custom designed additional icons that use a similar look.